### ISSUE TITLE
Quantization of video action recognition model

### ISSUE THREAD

**Issue by MrOCW** [microsoft/onnxruntime#10422]
Hi,

I am looking at quantization of a SlowFast model. The input, according to netron is 1x1x3x32x256x256. May I know if it is possible to perform PTQ on the model?

Do I just implement the CalibrationDataReader to have the same preprocessing steps as the training phase such that the calibration data has the same input dimensions as the onnx model?

Thanks!

**Comment by yufenglee** [comment]
Yes, please refer the example here: https://github.com/microsoft/onnxruntime-inference-examples/tree/main/quantization/image_classification

**Comment by MarvS95** [comment]
Hi @MrOCW 

I'm also trying to make PTQ with the SlowFast Network in order to compile it for inference on an FPGA. My approach is different as I try to use the quantizer from Vitis AI. At the moment I'm struggling with the input for the slow pathway [1, 3, 8, 256, 256] and the fast pathway  [1, 3, 32, 256, 256]. Does your model only have one input? Have you made any progress so far with the quantization?

**Comment by snnn** [comment]
This issue has been automatically closed as 'not planned' because it has been marked as 'stale' for more than 30 days without activity. If you believe this is still an issue, please feel free to reopen it.
